app/views/blazer/dashboards/show.html.erb

```diff
@@ -25,14 +27,32 @@
   </p>
 <% end %>
 
-<%= render partial: "blazer/variables", locals: {action: dashboard_path(@dashboard)} %>
-
 <% @queries.each_with_index do |query, i| %>
+  <% blazer_group = BlazerGroup.find(query.blazer_group_id).try(:name).try(:to_sym) %>
```

Comment:
Find will raise an exception if the id is not there.  So the `try(:name)` here at least is unnecessary.  Don't overly use trys as that's a crutch and can lead to maintenance issues.

Reply:
Good point. I'm requiring blazer_group_ids and names anyway for queries and dashboards so it's definitely unnecessary, I'll remove the trys.